docs/userguide/networking/overlay-security-model.md

```diff
@@ -0,0 +1,53 @@
+<!--[metadata]>
++++
+title = "Docker 1.12 overlay network security model"
+description = "Docker 1.12 Overlay Network Security Model"
+keywords = ["network, docker, documentation, user guide, multihost, swarm mode", "overlay"]
+[menu.main]
+parent = "smn_networking"
+weight=-2
++++
+<![end-metadata]-->
+
+## Note on Docker 1.12 Overlay Network Security Model
+
+The security model for Docker Engine swarm mode is designed to prevent the
+compromise of a single node from compromising the entire cluster.  This is
+different from other distributed systems where the compromise of a single
+machine immediately grants an attacker access to a central shared data store or
+cluster-management APIs.
+
+Two key architectural components provide the enhanced security of swarm mode:
+
+* **Unidirectional manager to worker communication model.**  Swarm mode worker
+nodes are dumb agents that execute tasks assigned to them by a manager node.
+Workers can never make requests to managers.  Therefore the compromise of a
+worker host machine does not allow an attacker to read or alter the swarm
+state, which is known only to managers.
+
+
+* **Workers connect to overlay networks on a need-to-know basis.** Only the nodes that run services employing an overlay network can access other nodes on the network.
+
+    For example, imagine you have a 100-node cluster that runs multiple
+    services. You constrain the microsevice that provides credit card payments
+    to five nodes that particpate in an overlay network called
+    `payment-card-net`.
+
+    In this case, only the five nodes on the `payment-card-net` exchange overlay
```

Comment:
@mgoelzer , please verify this behavior. Wouldn't at least one manager need to be in that group. Meaning managers know about overlay in addition to any participating nodes?